package.json

```diff
@@ -86,6 +86,7 @@
     "prop-types": "^15.6.0",
     "puppeteer": "^1.5.0",
     "qrcode": "^1.0.0",
+    "qs": "^6.5.2",
```

Comment:
qs stringifies objects to query strings and can also parse query strings into objects.